.golangci.yml

```diff
@@ -93,6 +93,7 @@ linters-settings:
       - G402  # TLS MinVersion too low
       - G504  # Import blocklist: net/http/cgi
       - G601  # Implicit memory aliasing in for loop (false positives)
+      - G115  # integer overflow conversion (TODO: verify these)
```

Comment:
same as https://github.com/moby/moby/issues/48358 . Doesn't look like anything critical here, but we could use some pattern for avoiding overflow.